en/embedding.html

```diff
@@ -277,6 +393,241 @@ <h2 id="examples">Examples</h2>
+      <th>Description</th>
+      <th>Type</th>
+      <th>Default</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>transformer-model</td>
+      <td>One</td>
+      <td>Accepts <em>path</em>, <em>url</em> and <em>model-id</em> attributes. Use to point to the transformer ONNX model file</td>
+      <td>string</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td>tokenizer-vocab</td>
+      <td>One</td>
+      <td>Accepts <em>path</em>, <em>url</em> and <em>model-id</em> attributes. Use to point to the <em>vocab.txt</em> file with valid wordpiece tokens</td>
+      <td>string</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td>max-tokens</td>
+      <td>One</td>
+      <td>The maximum number of tokens allowed in the input</td>
+      <td>integer</td>
+      <td>512</td>
+    </tr>
+    <tr>
+      <td>transformer-input-ids</td>
+      <td>One</td>
+      <td>The name or identifier for the transformer input IDs</td>
+      <td>string</td>
+      <td>input_ids</td>
+    </tr>
+    <tr>
+      <td>transformer-attention-mask</td>
+      <td>One</td>
+      <td>The name or identifier for the transformer attention mask</td>
+      <td>string</td>
+      <td>attention_mask</td>
+    </tr>
+    <tr>
+      <td>transformer-token-type-ids</td>
+      <td>One</td>
+      <td>The name or identifier for the transformer token type IDs</td>
+      <td>string</td>
+      <td>token_type_ids</td>
+    </tr>
+    <tr>
+      <td>transformer-output</td>
+      <td>One</td>
+      <td>The name or identifier for the transformer output</td>
+      <td>string</td>
+      <td>last_hidden_state</td>
+    </tr>
+    <tr>
+      <td>transformer-start-sequence-token</td>
+      <td>One</td>
+      <td>The start of sequence token</td>
+      <td>numeric</td>
+      <td>101</td>
+    </tr>
+    <tr>
+      <td>transformer-end-sequence-token</td>
+      <td>One</td>
+      <td>The start of sequence token</td>
+      <td>numeric</td>
+      <td>102</td>
+    </tr>
+    <tr>
+      <td>pooling-strategy</td>
+      <td>One</td>
+      <td>How the output vectors of the ONNX model is pooled to obtain a single vector representation. Valid values are <em>mean</em>mean and <em>CLS</em></td>
+      <td>string</td>
+      <td>mean</td>
+    </tr>
+    <tr>
+      <td>normalize</td>
+      <td>One</td>
+      <td>A boolean indicating whether to normalize the output embedding vector to unit length (length 1). Useful for <code>prenormalized-angular</code> 
+      <a href="reference/schema-reference.html#distance-metric">distance-metric</a></td>
+      <td>boolean</td>
+      <td>false</td>
+    </tr>
+    <tr>
+      <td>onnx-execution-mode</td>
+      <td>One</td>
+      <td>Low level ONNX execution model. Valid values are "parallel" or "sequential".
+         See <a href="stateless-model-evaluation.html#onnx-inference-options">onxx inference options</a></td>
+      <td>string</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>onnx-interop-threads</td>
```

Comment:
Consider moving ONNX configuration to a shared section

Reply:
thanks, done